tasks/utilities/task_compare_assemblies.wdl

```diff
@@ -21,25 +26,42 @@ task compare_assemblies {
                 if line.startswith(">"):
                     continue
                 seq += line.strip()
+            seq = seq.replace("N","")  # ignore uncalled bases
             return len(seq)
     
+    # File comparison
     denovo_base_count = count_bases("~{assembly_denovo}")
     consensus_base_count = count_bases("~{assembly_consensus}")
 
-    if denovo_base_count > consensus_base_count:
+
+    if denovo_base_count >= consensus_base_count:
         input_file = "~{assembly_denovo}"
+        with open("NUMBER_ATCG", "w") as number_ATCG:
+          number_ATCG.write(str(denovo_base_count))
     else:
         input_file = "~{assembly_consensus}"
+        with open("NUMBER_ATCG", "w") as number_ATCG:
+          number_ATCG.write(str(consensus_base_count))
 
     with open("~{samplename}_highest.fasta", 'w') as out_f:
```

Comment:
this section seems a bit inefficient to me. Could you not just say `~{samplename}_highest.fasta = input_file`? Then you don't have to iterate through the file.

Reply:
Also, this task isn't being called in the workflow. Maybe I'm going over this too soon and you're still working on it though 😂

Reply:
You're right. :) I wrote a lot of tasks that in the end I didn't end up using, like the bcftools one to call consensus sequences. I'm actually not sure if I should remove everything or leave them be in the case they are useful in the future

Reply:
I'll be removing the unused tasks that were included in this PR :)